Assets/MRTK/SDK/Features/UX/Scripts/ScrollingObjectCollection/README.md

```diff
@@ -1,5 +1,5 @@
 # Scrolling object collection
-![](../../../../../Documentation/Images/ScrollingCollection/MRTK_UX_ScrollingCollection_Main.jpg)
+![](../../../../../../../Documentation/Images/ScrollingCollection/MRTK_UX_ScrollingCollection_Main.jpg)
```

Comment:
This makes me wonder if going forward, we should just put images near where their content - i.e. the journey to the root here is a bit weird.

Anyways, this is a comment out loud just curious about what folks think for future things, not actually suggesting a change for this particular one.

Reply:
This README is actually going to be deleted. Its the README that comes inside experimental folder in the project for experimental stuff.

Reply:
But for experimental stuff, maybe we should only ask contributors to add READMEs to the actual Documentation folder structure outside the project.